index.html

```diff
@@ -104,6 +104,12 @@ <h1 class="heading">
         </div>
       </section>
 
+      <!-- The Modal -->
+      <section
+        id="myModal"
+        class="modal modal-dialog modal-dialog-centered modal-dialog-scrollable"
+      ></section>
+
       <!-- Works/projects section-->
       <section class="grid-container">
         <h2 class="heading-container heading-one" id="portfolio">
```

Comment:
- [ ] The works section should be created dynamically using the information stored in that JavaScript object as requested in the project requirements. Please update the works section and make sure that all of the HTML in that section is created when the page loads. `(You should delete the works section from the main page and create it in the javascript file dynamically).`

![image](https://user-images.githubusercontent.com/83803414/189145703-eb9984b4-b63f-48c9-b5ff-8633d0dddb13.png)